tests/container_test.rs

```diff
@@ -26,13 +26,7 @@ pub mod common;
 use crate::common::*;
 
 async fn list_containers_test(docker: Docker) -> Result<(), Error> {
-    let image = if cfg!(windows) {
-        format!("{}hello-world:nanoserver", registry_http_addr())
-    } else {
-        format!("{}hello-world:linux", registry_http_addr())
-    };
-
-    create_container_hello_world(&docker, "integration_test_list_containers").await?;
+    let image = create_container_hello_world(&docker, "integration_test_list_containers").await?;
```

Comment:
example of simplifying test by having `create_...` functions return the identifier of the resource they created